kubeflow/hack/create_context.sh

```diff
@@ -6,24 +6,18 @@
 # PROJECT=<PROJECT> REGION=<region> NAME=<NAME>
 #
 # TODO(jlewi): Support zonal clusters as well
-set -x 
+set -x
 
-# Default namespace to kubeflow
-NAMESPACE=${NAMESPACE:-kubeflow}
-echo Checking if context ${NAME} exists 
-
-kubectl config use-context ${NAME}
-
-RESULT=$?
-
-if [ ${RESULT} -eq 0 ]; then
-echo kubeconfig context ${NAME} already exists
-exit 0
-fi
+# Delete the existing kubeconfig, because it may be an outdated
+# context to a deleted cluster with the same name.
+kubectl config delete-context ${NAME} || echo "Context ${NAME} doesn't exist, Rename the context step will create it"
```

Comment:
> Does a user really need to get the details of what is going to happen behind the scenes?
OR
Notifying a user about intentionally ignoring an error is sufficient?

We are going to change an external state (kubeconfig) that Kubeflow doesn't manage, sounds like we should better tell the user about it.

> Also, I feel maybe we should not use so we will create a new one. Because who is we does not seem clear.

That's a good point!